src/main/java/rs/hybridit/model/BookInstance.java

```diff
@@ -0,0 +1,72 @@
+package rs.hybridit.model;
+import java.util.Date;
+import javax.persistence.*;
+
+@Entity
+@Table(catalog = "dbhybridlibrary", name = "book_instances")
+public class BookInstance {
+
+    @Id
+    @GeneratedValue
```

Comment:
some ids uses IDENTITY as strategy and some default value, you should be consistent always